nvflare/private/fed/server/job_cmds.py

```diff
@@ -202,6 +211,60 @@ def clone_job(self, conn: Connection, args: List[str]):
             return
         conn.append_success("")
 
+    def authorize_ls_job(self, conn: Connection, args: List[str]):
+        if len(args) < 2:
+            conn.append_error("syntax error: missing job_id")
+            return False, None
+
+        if len(args) > 3:
+            conn.append_error("syntax error: too many arguments")
+            return False, None
+
+        return self.authorize_job(conn=conn, args=args[:2])
+
+    def ls_job(self, conn: Connection, args: List[str]):
+        job_id = conn.get_prop(self.JOB_ID)
+
+        if len(args) == 2:
+            conn.append_string("job\nworkspace\n\nSpecify the job or workspace dir to see detailed contents.")
+            return
+        else:
+            file = args[2]
+
+        engine = conn.app_ctx
+        try:
+            job_def_manager = engine.job_def_manager
+            if not isinstance(job_def_manager, JobDefManagerSpec):
+                raise TypeError(
+                    f"job_def_manager in engine is not of type JobDefManagerSpec, but got {type(job_def_manager)}"
+                )
+            with engine.new_context() as fl_ctx:
+                job_data = job_def_manager.get_job_data(job_id, fl_ctx)
+                if file.startswith("job"):
+                    file = file[3:]
```

Comment:
if decide to go with this, 
Suggest using a constant to replace "job" or "workspace".

And use len(CONSTANT) to replace hardcode 3. (or 9.)

Reply:
done